adi/fmcomms5.py

```diff
@@ -72,6 +73,7 @@ def __init__(self, uri=""):
         self._rxadc_chip_b = self._ctx.find_device("cf-ad9361-B")
         self._txdac_chip_b = self._ctx.find_device("cf-ad9361-dds-core-B")
         rx_tx.__init__(self)
+        libad9361.fmcomms5_multichip_sync(self._ctx, 2)
```

Comment:
I think this should be 2->3  since we want binary 11. This was a bug in the tests for pylibad9361

Reply:
Implemented through commit 205930d.